addons/mail/static/src/models/autocomplete_input_view.js

```diff
@@ -11,6 +11,17 @@ registerModel({
         'discussViewOwnerAsMobileAddItemHeader',
         'messagingMenuOwnerAsMobileNewMessageInput',
     ]],
+    recordMethods: {
+        /**
+         * @param {FocusEvent} ev
+         */
+        onFocusin(ev) {
+            if (this.chatWindowOwnerAsNewMessage) {
+                this.chatWindowOwnerAsNewMessage.onFocusInNewMessageFormInput(ev);
```

Comment:
I wonder if we actually want to keep the method, or move its code here?
But otherwise diff for the handler in this PR is ok.

Reply:
I prefer not doing it now, to keep PR strictly focused on intend of moving code from component to models.
Also there are some drawbacks with removing `onFocusInNewMessageFormInput` and dump code here, namely `AutocompleteInputView` code can become quite hard to maintain. Some POC versions had this issue.

Reply:
Ok then.